editor/osm_auth.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "std/string.hpp"
+#include "std/utility.hpp"
+
+namespace osm
+{
+
+using TKeySecret = pair<string /*key*/, string /*secret*/>;
```

Comment:
It's a cross-class type, and it's actively used in osm namespace. I propose to leave it as is.